packages/fiori/src/fcl-utils/FCLLayout.ts

```diff
@@ -1,22 +1,37 @@
-const getLayoutsByMedia = () => {
+import { MEDIA } from "../FlexibleColumnLayout.js";
+
+type LayoutConfiguration = {
+	[key in MEDIA]: {
+		[key: string]: {
```

Comment:
as discused in #6309 

For me better solution here is to import FCLLayout from types. It will prevent us from adding unused layouts.

```ts
type LayoutConfiguration = {
	[key in MEDIA]: {
		[key in FCLLayout]
	};
};
```

It's complaining for ThreeColumnsStartExpanded. Since layout accepts values from FCLLayout enum only and ThreeColumnsStartExpanded is not part of it, it will fallback to the initial (default) value which is OneColumn.

I think that we can remove it.

Reply:
Removing it. ✅